symplyphysics/laws/optics/radius_newton_ring_from_wavelength_order_radius_curvature_refractive_index.py

```diff
@@ -0,0 +1,55 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    angle_type
+)
+
+# Description
+## Newton's rings are annular interference maxima and minima that appear around the point of contact between
+## a convex lens and a plane—parallel plate when light passes through the lens and plate.
+## The order of interference indicates the number of whole wavelengths. When the difference in the path of the
+## beam reflected from the convex surface of the lens at the glass-air boundary and the beam reflected from the
+## plate at the air-glass boundary is equal to an entire wave "n", then these two waves hit the observation
+## point with the same faces and demonstrate interference.
+
+## Law is: r = sqrt(k * R * L / n), where
+## r - film thickness,
+## k - order of interference,
+## R - radius of curvature of lens
+## L - wavelength,
+## n - refractive index of lens.
```

Comment:
Shouldn't it also be somehow dependent on the medium in the gap between lens and a plate?